.circleci/config.yml

```diff
@@ -38,7 +38,6 @@ jobs:
           docker-compose version
           nodejs -v
           jq --version
-          package_cloud version
```

Comment:
Is the removal of `package_cloud version` intentional?

Reply:
Was getting an error with not found - so just trying to debug it.

Reply:
Moving PR into draft whilst I get the circleCI to pass...
This is the only repo that uses the package_cloud.sh that had that package_cloud version call - so just seeing what's going on...